### DISCUSSION TITLE
O-RAN Split 7.2 with virtual RAN

### DISCUSSION THREAD

**Discussion by jmarkotic** [srsran/srsRAN_Project#187] [General Help] [answered]
Hi,
i am looking at tutorial for srsRAN o-ran 7.2 split guide.
I am assuming not possible, but still going to ask ... is it possible to leverage zmq based virtual RF driver and replace RU/antenna when testing with 7.2 split setup ?

**Comment by andrepuschmann** [comment, marked as answer]
Hey @jmarkotic - I am afraid that's not possible. The current ZMQ virtual RF driver is sending/receiving baseband IQ samples and not OFH frames. 

But I guess your question is more if it is possible to leverage the OFH interface to run a 7.2 split without an actual RU. And the answer here is yes - in theory. If there was a software implementation of an RU that would be possible. Unfortunately we are not aware of any.

**Reply by jmarkotic** [reply]
thanks @andrepuschmann ! Yes, that;s the reason i asked - to be able to test 7.2 split arch without physical O-RU.
